packages/sanity/src/core/releases/tool/BundleDetail.tsx

```diff
@@ -0,0 +1,104 @@
+import {ArrowLeftIcon, PublishIcon, StarIcon} from '@sanity/icons'
```

Comment:
This file was changed for `ReleaseDetail` https://github.com/sanity-io/sanity/blob/corel/packages/sanity/src/core/releases/tool/detail/ReleaseDetail.tsx

We should remove this and update the tests